### DISCUSSION TITLE
How to create and map a command to open links in split screen (in Edge only, probably)?

### DISCUSSION THREAD

**Discussion by toraritte** [gdh1995/vimium-c#1191] [Ideas]
Edge has an "Open link in split screen window" in the context menu when right clicking on a link that I've been using quite often recently. I searched through the issues and discussions, but didn't find anything, so it is either too straightforward (and I missed this in the docs) or this hasn't been asked yet. (Or it was asked but I missed it.)

Thank you for this awesome tool!

**Comment by gdh1995** [comment]
Um, sorry but usually such browser-specified built-in features are not available for web-extensions, which means (I think) Vimium C can not tell MS Edge to open a side bar.
